lesson-2/Lee - Lesson 2/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Don't forget to close this `<html>` tag with a `</html>` at the end of the document